data/brands/shop/doityourself.json

```
@@ -211,16 +211,27 @@
       "displayName": "Bricoman",
       "id": "bricoman-5e1049",
       "locationSet": {
-        "include": ["fr", "it", "pl"]
+        "include": ["fr", "pl"]
       },
       "tags": {
         "brand": "Bricoman",
         "brand:wikidata": "Q2925142",
-        "brand:wikipedia": "it:Bricoman",
         "name": "Bricoman",
         "shop": "doityourself"
       }
     },
+    {
+      "displayName": "Tecnomat",
+      "locationSet": {
+        "include": ["it"]
+      },
+      "tags": {
+        "brand": "Bricoman",
+        "brand:wikidata": "Q2925142",
```

```suggestion
        "brand": "Tecnomat",
        "brand:wikidata": "Q111716870",
```
Tecnomat has its own Wikidata item:
https://www.wikidata.org/wiki/Q111716870